articles/fleet-desktop-says-hello-world.md

```diff
@@ -0,0 +1,32 @@
+# Fleet Desktop Says “Hello, World!”
+
+![Fleet Desktop v1 preview](../website/assets/images/articles/fleet-desktop-says-hello-world-cover.png)
+
+We are thrilled to announce that Fleet Desktop is out of Beta! Over the past few months, we’ve dogfooded Fleet Desktop internally and validated its usability and scalability. It is now ready for production deployment across your host fleet.
+
+## What is Fleet Desktop?
+
+Fleet Desktop is a menubar icon available on macOS, Windows, and Linux. At its core, Fleet Desktop gives your end users visibility into the security posture of their machine. This unlocks two key benefits:
+
+__Self Remediation.__ When a host is found to be out of compliance with one or more policies configured in Fleet, Fleet Desktop will display the red error icon and list the number of failing policies, as seen in the screenshot above.
```

Comment:
Should "more policies configured in Fleet," read "more policies configured in fleetctl,"

Reply:
I think policies are configurable in both the fleetctl and the fleet UI, so I think it is more accurate to say "Fleet" in general here, rather than fleetctl specifically. WDYT?